src/components/strategies/overview/StrategyFilterSort.tsx

```diff
@@ -18,24 +17,24 @@ export enum EnumStrategySort {
   RoiAscending,
   RoiDescending,
 }
+
 export const strategySortMapping: Record<EnumStrategySort, StrategySort> = {
   [EnumStrategySort.Recent]: 'recent',
   [EnumStrategySort.Old]: 'old',
   [EnumStrategySort.PairAscending]: 'pairAsc',
   [EnumStrategySort.PairDescending]: 'pairDesc',
-  [EnumStrategySort.RoiAscending]: 'roiAsc',
-  [EnumStrategySort.RoiDescending]: 'roiDesc',
+  // default to "trades"
+  [EnumStrategySort.RoiAscending]: 'trades',
+  [EnumStrategySort.RoiDescending]: 'trades',
```

Comment:
Could you update the strategySort here and create a LS migration instead? That way if RoiAscending or RoiDescending was selected -> default to trades